README.txt

```diff
@@ -439,7 +439,7 @@ Solaris 11:
 
 Debian/Ubuntu:
 
-  # apt-get install sun-java6-javadb
+  # apt-get install sun-java8-javadb
 
```

Comment:
I don't think there's any package with that name in Debian or Ubuntu. There seems to be a libderby-java package, though.